### PR TITLE
Quote view names

### PR DESCRIPTION
**Pull request by calebthompson**

### CODE REVIEW

**Comment by calebthompson** [comment]
Closes #172

File: lib/scenic/adapters/postgres/views.rb [hunk 1]
```diff
@@ -54,6 +54,11 @@ def to_scenic_view(result)
             materialized: result["kind"] == "m",
           )
         end
+
+        def pg_identifier(name)
+          return name if name =~ /^[a-zA-Z_][a-zA-Z0-9_]*$/
```
Comment (derekprior) [inline lib/scenic/adapters/postgres/views.rb]: Can we not safely quote the identifier in any case?

**Comment by calebthompson** [comment]
I tried removing this and it ended up causing double quoting because we'd quote on the way out of the db and when it goes in. I think it's only code, but it does break the scheme dump.

File: lib/scenic/adapters/postgres/views.rb (hunk 1 above)
Comment (calebthompson) [inline lib/scenic/adapters/postgres/views.rb]: Removing this line caused double quoting in in-memory representations of the names (failing tests) and in the SQL dump (failing more tests, and breaking things). I wasn't able to fix that in ~an hour, so I reverted my attempts.

**Pull request merged**